contracts/dmarket.js

```diff
@@ -0,0 +1,1285 @@
+/* eslint-disable no-await-in-loop */
+/* global actions, api */
+
+const UTILITY_TOKEN_SYMBOL = 'BEE';
+
+// 30 days in seconds
+const MAX_EXPIRATION_SECS = 2592000;
+// fee in UTILITY TOKEN required to create a new pair
+const PAIR_CREATION_FEE = '500';
+
+const CONTRACT_NAME = 'dmarket';
+
+actions.createSSC = async () => {
+  const tableExists = await api.db.tableExists('pairs');
+
+  if (tableExists === false) {
+    await api.db.createTable('pairs', ['symbol', 'precision', 'allowedSymbols']);
+    await api.db.createTable('buyBook', ['symbol', 'pair', 'account', 'priceDec', 'expiration', 'txId']);
+    await api.db.createTable('sellBook', ['symbol', 'pair', 'account', 'priceDec', 'expiration', 'txId']);
+    await api.db.createTable('tradesHistory', ['symbol', 'pair']);
+    await api.db.createTable('metrics', ['symbol', 'pair']);
+
+    // default global pairs
+    const pairs = [
+      {
+        pair: 'BEE',
+        precision: 8,
+        allowedSymbols: true, // true - all tokens are allowed
+      },
+      {
+        pair: 'SWAP.BTC',
```

Comment:
I think we should use terminology consistent with https://www.investopedia.com/terms/c/currencypair.asp . In the terminology, price represents how much of the quote token is needed to purchase 1 unit of the base token. 

So that means this 'pair' should be renamed 'quoteToken'

Also suggest renaming 'allowedSymbols' to  'allowedBaseTokens'